src/sponsors/models.py

```diff
@@ -56,6 +56,12 @@ class Sponsor(ConferenceRelated):
             "fallback when the SVG file is not available."
         ),
     )
+    order = models.IntegerField(
+        verbose_name=_('order'),
+        max_length=10,
+        null=True,
+        blank=True,
+    )
```

Comment:
- `max_length` will not take any effect for the numeric field. If you'd like to constrain the range of the value, use [`validators`](https://docs.djangoproject.com/en/dev/ref/validators/#maxvaluevalidator) instead.
- Would using only positive value make more sense? (e.g. use `PositiveIntegerField` instead)
- IMO it's not easy to realize the purpose of this field simply by its name. Consider adding `help_text` here.
- What happened if there is any sponsor instance with this field to be null? Would it be at the start or the end of the returned list (and is that what you expected)?